### PR TITLE
fix: removing che theia from troubleshooting slow workspaces (rhdevdocs-4580)

### PR DESCRIPTION
**Pull request by deerskindoll**
## What does this pull request change?
removing che theia from troubleshooting slow workspaces

## What issues does this pull request fix or reference?
https://issues.redhat.com/browse/RHDEVDOCS-4572

## Specify the version of the product this pull request applies to
devspaces 3.6, eclipse che 7.63

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/main/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [x] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Review by ibuziuk — COMMENTED** [review]
LGTM @azatsarynnyy @RomanNikitenko folks, not really sure though if the same instructions are valid for che-code

**Comment by azatsarynnyy** [comment]
> LGTM @azatsarynnyy @RomanNikitenko folks, not really sure though if the same instructions are valid for che-code

@ibuziuk What is related to the `Reducing the number of public endpoints` section is valid for Che-Code as well
https://github.com/eclipse-che/che-plugin-registry/blob/aea518ad4b8773534569a6bc8c640cf207e15063/che-editors.yaml#L212-L232

But `CDN configuration` is related to Che-Theia only. So, it can be removed.
@deerskindoll ^^

Also, I think that the `Improving workspace runtime performance` section should be revised as it mentions `meta.yaml` plugin descriptors that are not relevant for plugin-registry anymore.

**Review by ibuziuk — COMMENTED** [review]
@deerskindoll the following section should be removed:

```
Optimizing workspace plugins::
+
_Role: User_
+
When selecting various plugins, each plugin can bring its own sidecar container, which is an OCI image. {orch-name} pulls the images of these sidecar containers.
+
Reduce the number of plugins, or disable them to see if start time is faster. See also xref:administration-guide:caching-images-for-faster-workspace-start.adoc[].
Reducing the number of public endpoints::
```

**Comment by ibuziuk** [comment]
@deerskindoll CDN configuration should be removed

```
CDN configuration::
+
The IDE editor uses a CDN (Content Delivery Network) to serve content. Check that the content uses a CDN to the client (or a local route for offline setup).
+
To check that, open Developer Tools in the browser and check for `vendors` in the *Network* tab. `vendors.<random_id>.js` or `editor.main.*` should come from CDN URLs.
```

**Review by azatsarynnyy — APPROVED** [review]
LGTM

**Review by svor — APPROVED** [review]
LGTM

File: modules/end-user-guide/pages/troubleshooting-slow-workspaces.adoc [hunk 1]
```diff
@@ -70,52 +55,31 @@ Ensure the CPU settings in the devfile definition, `devfile.yaml`, are correct:
...
-      memoryLimit: "512Mi" <1>
-  extensions:
-    - https://link.to/vsix
 ----
-<1> Specifies the memory limit for the plugin.
-+
-In the devfile definition (`devfile.yaml`):
-+
 [source,yaml]
 ----
```
Comment (aireilly) [inline modules/end-user-guide/pages/troubleshooting-slow-workspaces.adoc:75]: This code block is not closed.

Comment (deerskindoll) [reply modules/end-user-guide/pages/troubleshooting-slow-workspaces.adoc:75]: it got deleted and won't appear in the new version of the document

Comment (aireilly) [reply modules/end-user-guide/pages/troubleshooting-slow-workspaces.adoc:75]: Are you sure? I see an unclosed codeblock here.

image

Comment (deerskindoll) [reply modules/end-user-guide/pages/troubleshooting-slow-workspaces.adoc:75]: you're right. that one stays.

**Pull request merged**